tests/bearlib/languages/documentation/DocstyleDefinitionTest.py

```diff
@@ -73,7 +92,13 @@ def test_load(self):
         self.assertEqual(result.docstyle, "default")
         self.assertEqual(result.markers, (('"""', '', '"""'),))
 
+        self.assertEqual(result.metadata, self.dummy_metadata)
+        self.assertEqual(result.metadata.param_start, ":param ")
+        self.assertEqual(result.metadata.param_end, ":")
+        self.assertEqual(result.metadata.return_sep, ":return:")
```

Comment:
I think the last three asserts are not needed, as if the tuple does equal the items do too